content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -165,6 +181,7 @@ dual-stack.)
 
    You can validate this behavior by using kubectl to inspect an existing service.
 
+<<<<<<< HEAD
```

Comment:
This looks like a manual merge gone wrong; please fix it.

Reply:
Apologies, I must have missed this during one of the rebases I had to do.